### ISSUE TITLE
multi-gpu distributed and LR

### ISSUE THREAD

**Issue by op-newman** [lyuwenyu/RT-DETR#144]
In paddle version of detr training , I have only one card, so how do I adjust the LR, I queried that the original version of rtd detr was 4XA100 trained I now have only one 3090, so how do I adjust the learning rate, also in pytorch version I have the same problem, for the adjustment of the learning rate I didn't understand. I want to train my own data and I have only one GPU, how do I set up the LR

**Comment by op-newman** [comment]
Also in my detailed testing I found that rtdetr doesn't seem to be very friendly with small data I had a very awkward scene when testing with the cat data provided by mmdet
![image](https://github.com/lyuwenyu/RT-DETR/assets/115713675/c201ae6b-93b3-4da5-97a6-1d7455499e4f)
